### ISSUE TITLE
SFW-content only

### ISSUE THREAD

**Issue by brettneese** [HackathonHackers/code-of-conduct#46]
Similar to #44, I feel as if we should make it incredibly explicit under "Da Rules" that content in HH should not be "not safe for work."

Something like:

- Safe-for-work content only. Hackathon Hackers is a student and professional community. The line between SFW and NSFW content is different for different workplaces and work cultures, but if browsing certain content on Hackathon Hackers at work could get a member fired, it should not be in Hackathon Hackers.

This would prevent a lot of quibbling over edge cases.

**Comment by rubinovitz** [comment]
Interesting. In response to recent events I created #44 , but we haven't decided whether to merge it or not. I think one of the big issues with the post we're referring to was that it was NSFW, but I'm unsure how to uniformly enforce NSFW across different work cultures across the world. Is there a more precise definition than guessing what is safe at everyone's work?

**Comment by Gchorba** [comment]
The problem with this is there a sliding scale of what is SFW. Not a fan of this vague definition, so I lean towards not doing this.

**Comment by brettneese** [comment]
@rubinovitz I agree that's a hard problem. So not sure. But seems like a nice blanket - if one has to ask if it's "safe for work," is it?

**Comment by wongmjane** [comment]
I think the [Appendix A definition](https://github.com/HackathonHackers/code-of-conduct#appendix-a-definitions) on harassment should cover much enough of what constituties as "NSFW" (e.g. Unwelcome sexual attention, including gratuitous or off-topic sexual images or behaviour).
